<launch>
  <arg name="computer_name" value="usl" />
<!--   <param name ="/use_sim_time" value="true"/> -->
  <node  pkg="projection" name="line_basler_os_projection_node" type="line_projection_node" output="screen">
    <remap from="/line_image1" to="/basler/line_image1"/>
    <remap from="/line_image2" to="/basler/line_image2"/>
    <remap from="/line_image3" to="/basler/line_image3"/>
    <remap from="/line_image4" to="/basler/line_image4"/>
    <remap from="/edge_cloud_in" to="/os1_cloud_node/points/rotated/with_edges/plane"/>
    <remap from="/image_in" to="/pylon_camera_node/image_raw"/>
    <param name ="result_str" value="/home/$(arg computer_name)/catkin_ws/src/pbpc-cal/calibration_backend_optimization/debug_data/draw/C_T_L_basler_os.txt"/>
    <param name ="cam_config_file_path" value="/home/$(arg computer_name)/catkin_ws/src/pbpc-cal/config/basler.yaml"/>
    <param name="base_folder_name" value="/home/$(arg computer_name)/projection_results/basler_lineprojection_node_os/"/>
    <param name ="target_config_file_path"
           value="/home/$(arg computer_name)/catkin_ws/src/pbpc-cal/config/target_3.yaml"/>
    <param name="csv_file_name" value="/home/$(arg computer_name)/catkin_ws/src/pbpc-cal/calibration_backend_optimization/debug_data/draw/PoseVsRepErr_basler_os.csv"/>
    <param name="camera_name" value="basler"/>
  </node>
</launch>
